regolith/helpers/a_proposalhelper.py

```diff
@@ -28,70 +23,57 @@ def subparser(subpi):
     subpi.add_argument("name", help="A short but unique name for the proposal",
                        default=None
                        )
-    subpi.add_argument("-amt", "--amount", help="value of award",
+    subpi.add_argument("amount", help="value of award",
                        default=None
                        )
-    subpi.add_argument("-a", "--authors",
-                       help="Other investigator names"
+    subpi.add_argument("authors", help="Other investigator names"
                        )
-    subpi.add_argument("--begin_day", nargs="+",
-                       help="The start day for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("currency", help="Currency in which amount is specified."
+                                        " Typically $ or USD"
                        )
-    subpi.add_argument("--begin_month", nargs="+",
-                       help="The start month for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("due_date", help="The due date for the proposal in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--begin_year", nargs="+",
-                       help="The start year for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("duration", help="Duration of proposal in years"
                        )
-    subpi.add_argument("--end_date", nargs="+",
-                       help="The end date for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("submitted_date", help="Day on which the proposal is submitted "
+                                              "in format YYYY-MM-DD"
                        )
-    subpi.add_argument("--call_for_proposals", nargs="+",
-                       help="Url for call for proposals"
+    subpi.add_argument("pi", help="Name of principal investigator"
                        )
-    subpi.add_argument("--cpp_info", nargs="+",
-                       help="Extra information needed for building current and"
-                            "pending form"
+    subpi.add_argument("title", help="Actual title of the proposal"
                        )
-    subpi.add_argument("-c", "--currency",
-                       help="Currency in which amount is specified."
-                            " Typically $ or USD"
+    subpi.add_argument("-b", "--begin_date", nargs="+",
+                       help="The start date for the proposed grant in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--submitted_date",
-                       help="Day on which the proposal is submitted in format YYYY-MM-DD"
+    subpi.add_argument("--cpp_flag", nargs="+",
+                       help="Current and pending form (true or false)"
                        )
-    subpi.add_argument("--due_date", nargs="+",
-                       help="The due date for the proposal in format YYYY-MM-DD."
+    subpi.add_argument("--cpp_agencies", nargs="+",
```

Comment:
no need to hvae the schema structure leak into the arg, so I would just call this ``other_agencies``